docs/en/guides/pipeline-integrations/administration/config-staging-and-rollout.md

```diff
@@ -6,3 +6,489 @@ lang: en
+
+### Setting up your studio config repository
+
+Before you do anything else, you need to create a studio config repository. This section shows how to take an existing toolkit configuration and creating a git repository from that.
+
+First, you need to go to your git server and create a repository. This process may be different, depending on how you run things. If you are using something like github, you would start a web browser and navigate to github. If you have access to the server you may do something like  `git init --bare`. In our example, we assume that the git repository you create is called  `username@someserver.com:/studio_config.git`.
+
+Now move the  `config`  folder of the project you want to use to seed your repo with into a  `config.bak`  location:
+```shell
+cd /project_configs/studio_config
+mv config config.bak
+```
+Clone your initialized git repository into the  `config`  location of your project that you want to base the studio config on. Once you have run the clone command, you will have an empty  `config folder`  which is also a git repository:
+```shell
+cd /project_configs/studio_config
+git clone username@someserver.com:/studio_config.git config
+```
+Copy all the files from your  `config.bak`  location back into the  `config`  folder. Once done, you can delete the empty  `config.bak`  folder. Your config files are now inside the git repository and we need to add them, commit them and push them to the server. But before doing that, we need to do some house keeping to handle some toolkit system files correctly. In the  `config`  folder, create a  `.gitignore`  file and add the following lines to it:
+```shell
+install_location.yml
+pipeline_configuration.yml
+```
+Now git will ignore those two system files. We can go ahead and add the rest of the files in the configuration, commit them and push them to the studio repository.
+```shell
+git add --all
+git commit -am "initial commit of our studio config!"
+git push
+```
+### Creating a new project from git
+
+When you create a new project, simply specify a valid git url when the setup process prompts you to enter the path to the configuration to use. Following our example above, we would enter  `username@someserver.com:/studio_config.git`. As part of the project setup process, Toolkit will clone this repository into the  `config`  folder of your new project configuration. This means that you can later on go into this config folder and run git commands. Note that any cloned pipeline configurations will also clone the git repository and will work seamlessly.
+
+### Making changes to a project
+
+Whenever you have made changes to your primary config, you can simply go to your  `config`  folder and then run  `git diff`,  `git commit`  and  `git push`. This would push your changes to the studio configuration that you cloned your project config from.
```

Comment:
```suggestion
Whenever you have made changes to your primary config, you can simply go to your `config` folder and then run `git diff`, `git commit` and `git push`. This would push your changes to the studio configuration that you cloned your project config from.
```